tests/openwisp2/settings.py

```diff
@@ -30,6 +30,8 @@
     # Only for developement
     'django_extensions',
     'drf_yasg',
+    # Other dependencies
+    'reversion',
```

Comment:
please add this also in the setup instructions in the README or users won't be able to replicate the setup correctly

Reply:
Done.